### DISCUSSION TITLE
Welcome to LycanthropyPlus Discussions!

### DISCUSSION THREAD

**Discussion by atomicAxolotl** [atomicAxolotl/LycanthropyPlus#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gshdg** [comment]
ls this version of the werewolf mod finished? May be releasedin the Chinese version of minecraft， where there are many players eager!

**Reply by MimzyFang** [reply]
it's far from being done.
unpack zip and put it in to your mods folder if you want to try it out it is forge 1.16.4
[mod.zip](https://github.com/atomicAxolotl/LycanthropyPlus/files/5980004/mod.zip)

**Reply by gshdg** [reply]
> it's far from being done.
> unpack zip and put it in to your mods folder if you want to try it out it is forge 1.16.4
> [mod.zip](https://github.com/atomicAxolotl/LycanthropyPlus/files/5980004/mod.zip)

What is in this l ink,

please? Forgive me fornot knowing much English

**Reply by MimzyFang** [reply]
in this zip is a mod jar I compiled from source you can do it yourself if you use an older commit the current is failing to build

don't worry my English is not that great either

**Reply by gshdg** [reply]
https://www.curseforge.com/minecraft/mc-mods/werewolves-become-a-beast
I found a new mod of werewolf

**Reply by gshdg** [reply]
Hey, buddy. It's been almost a year. I don't know how this mod is doing now, except for the howling moon of 1.7.10 and 1.12.2. I haven't played a more fun higher version of the werewolf mod.
